### ISSUE TITLE
ISSUE: Can't open my drawings after upgrading from 2.0.20 -> 2.0.21

### ISSUE THREAD

**Issue by Bobatev** [zsviczian/obsidian-excalidraw-plugin#1600]
**Your environment**
SYSTEM INFO:
	Obsidian version: v1.5.6
	Installer version: v1.5.3
	Operating system: Windows 11 Pro 
	Login status: logged in
	Catalyst license: insider
	Insider build toggle: on
	Live preview: on
	Base theme: dark
	Community theme: Blue Topaz v2023112601
	Snippets enabled: 29
	Restricted mode: off
	Plugins installed: 23
	Plugins enabled: 23
		1: Dataview v0.5.64
		2: Editing Toolbar v2.4.0
		3: Readwise Official v2.0.1
		4: Templater v2.2.1
		5: Text Format v2.5.1
		6: Tag Wrangler v0.6.1
		7: Image in Editor v2.2.6
		8: Emoji Toolbar v0.4.1
		9: Recent Files v1.3.10
		10: Homepage v3.7.0
		11: Pandoc Plugin v0.4.1
		12: Paste URL into selection v1.7.0
		13: Highlightr v1.2.2
		14: LanguageTool Integration v0.3.7
		15: Excel to Markdown Table v0.4.0
		16: Footnote Shortcut v0.1.3
		17: Multi-Column Markdown v0.9.1
		18: Smart Typography v1.0.18
		19: Paste image rename v1.6.1
		20: Underline v1.2.2
		21: Style Settings v1.0.7
		22: Heading Level Indent v1.0.3
		23: Excalidraw v2.0.21

**Describe the bug**
After I updated Excalidraw from 2.0.20 -> 2.0.21, I can open my drawings only in md view.

It says that the file is corrupted.

**To Reproduce**
1. Download enclosed file

[Skill 3 Refutation.excalidraw.md](https://github.com/zsviczian/obsidian-excalidraw-plugin/files/14342821/Skill.3.Refutation.excalidraw.md)

2. Open with Obsidian Excalidraw

Thanks a lot!

Bozhidar

P.S. I'm not sure that Excalidraw upgrade is the root cause of the issue as the problem appears after I restarted my laptop (no wind updates or new programs installed in between)

**Comment by zsviczian** [comment]
You mention "Drawings" in the title. Are multiple drawings impacted, or just this one?

**Comment by Bobatev** [comment]
Hey Zoltan!

You are awesome, my friend! Really, really thank you very much.

Frankly, I'm not a developer and at way below your level to understand your explanation.

Interestingly enough, I haven't touched this file since 10th of February, and it was working at that time. A possible explanation could be an issue with synchronization we were troubleshooting a couple of days ago with a guy from Obsidian team.

I don't know how many files are affected as I have 100's. I found 4 more corrupted but managed to restore them from a backup.

Anyways,

Thank you very much for the time and efforts. I'm really happy to have you with us and monthly donation is the smartest and best investment I've made for years. It would be a pleasure if I could be of any help. Just let me know.

Bozhidar
